metadata.json

```diff
@@ -38,7 +38,8 @@
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
         "10",
-        "11"
+        "11",
+        "12"
```

Comment:
At the moment, there is no Puppet release for Debian 12, so we will wait for the release before merging this PR.  This will allow us to test this is working and (more important in the long run) ensure we do not break it in the future.

Stay tuned!

Reply:
I only found this forum post https://groups.google.com/g/puppet-users/c/vlGQ8I9aBQg 

Unfortunatly i can't look into the mentioned jira ticket. Do you have more insight? 
Are they still planning to release an Debian 12 version? Seems like we are stuck with the one provided via the debian repos (7.23) at the moment and given that it has been almost 2 months since the release of debian 12...

To be honest i kinda hope they will still release it because the debian version uses a different folder structure which breaks some other modules we are using -_-

Reply:
I don't have more details, I just *know* that this is on a roadmap and will eventually happen.  In the meantime, if you are using Debian 12 nodes with the Debian 11 package (which seems to work from my home lab experience), I would recommend to adjust r10k config (or any system you use to manage your puppet modules) to point to your fork until the release happen and we can "officially" add support for Debian 12 to the module.

Reply:
Ah, haven't tried using the Debian 11 version. I'll give it a shot tomorrow.
Well if it's on the roadmap then I'll have my fingers crossed that it gets released soon 😃
Thanks